src/pages/home.jsx

```diff
@@ -14,48 +14,68 @@ class Home extends React.Component {
         this.state = {
             data: []
         };
+
+        this.handleContactRequestSent = this.handleContactRequestSent.bind(this);
     }
 
     componentDidMount() {
-        fetch('http://localhost:5000/api/users?page=1')
-            .then(res => {
-                return res.json();
-            })
-            .then(users => {
-                this.setState({ data: users.data });
-            });
+        const userId = (JSON.parse(atob(localStorage.jwt.split(".")[1]))).sub,
+            recommend = localStorage.userType === "employee" ? "jobseekers" : "employees";
+
+        fetch(`http://localhost/api/recommendations/${userId}/${recommend}`).then(res => {
+            return res.json();
+        }).then(users => {
+            this.setState({data: users});
+        });
+    }
+
+    handleContactRequestSent(id) {
```

Comment:
Not sure I understand this function

Reply:
If a contact request was successfully sent from the home page, the SwiperSlide component needs to be removed in order to not leave a gap when the Match component is hidden. This function accomplishes that by removing the user from the array of contact recommendations.

Will add clarifying comments.